### DISCUSSION TITLE
curly bracket in long url

### DISCUSSION THREAD

**Discussion by empthrichter** [YOURLS/YOURLS#3565] [Q&A] [answered]
I have a long URL with curly brackets included like http://domain.tld?keys={{value1}}-{{value2}}
After shortening the link the brackets in the long URL are lost: http://domain.tld?keys=value1-value2
Is there any way to keep the brackets?

**Comment by dgw** [comment, marked as answer]
Curly braces are part of the character set labeled "unsafe" for URLs per standards (at least historically). You should [urlencode](https://www.urlencoder.org/) your links before shortening.
